File: php/Siak.php
<?php

class Siak extends Human{

    private $asalUniv, $emailEdu;

    public function __construct($asalUniv = "", $emailEdu = ""){
        $this->asalUniv = $asalUniv;
        $this->emailEdu = $emailEdu;
    }

    public function setEmailEdu($emailEdu){
        $this->emailEdu = $emailEdu;
    }
    public function getEmailEdu(){
        return $this->emailEdu;
    }

    public function setAsalUniv($asalUniv){
        $this->asalUniv = $asalUniv;
    }
    public function getAsalUniv(){
        return $this->asalUniv;
    }

    public function __destruct(){
        
    }
}
?>

File: python/Siak.py
from Human import Human

class Siak(Human):
    
    def __init__(self):
        self.asalUniv = ""
        self.emailEdu = ""
    
    def setAsalUniv(self, asalUniv):
        self.asalUniv = asalUniv
    
    def setEmailEdu(self, emailEdu):
        self.emailEdu = emailEdu
    
    def getAsalUniv(self):
        return self.asalUniv
    
    def getEmailEdu(self):
        return self.emailEdu